Loop54.Tests.Shared/CallMethods.cs

```diff
@@ -67,23 +67,21 @@ public void GetRelatedEntitiesHasResults()
         }
 
         [Test]
-        [Ignore("Not released to HelloWorld engine yet")]
         public void GetComplementaryEntitiesHasResults()
         {
             //Should be a wheat flour
             var response = GetClient().GetComplementaryEntities(WrapRequest(new GetComplementaryEntitiesRequest("Product", "13")));
-            Assert.Greater(response.Results.Count, 0);
-            Assert.Greater(response.Results.Items.Count, 0);
+            Assert.AreEqual(response.Results.Count, 0); // Currently no complementary entities in HelloWorld engine
```

Comment:
Not great, but at least it checks that the API call does not fail - better than nothing